### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by andreasknoepfle**

### CODE REVIEW

File: spec/integration/dummy_app/config/environments/test.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
   # Enable/disable caching. By default caching is disabled.
   # Run rails dev:cache to toggle caching.
-  if Rails.root.join('tmp', 'caching-test.txt').exist?
+  if Rails.root.join('tmp/caching-test.txt').exist?
```
Comment (nickgnd) [inline spec/integration/dummy_app/config/environments/test.rb:19]: Why is this change required? Rubocop?

**Review by nickgnd — APPROVED** [review]
👍 Wonderful

File: spec/integration/dummy_app/config/environments/test.rb (hunk 1 above)
Comment (andreasknoepfle) [reply spec/integration/dummy_app/config/environments/test.rb:19]: Yes. Rubocop update.
